openssh/Patches/0002-PAM-Support.patch

```diff
@@ -31,16 +31,15 @@ index 4d1910fe..ac105537 100644
  
  	/* Standard Options */
  	if (options->num_host_key_files == 0) {
-@@ -1309,8 +1314,17 @@ process_server_config_line_depth(ServerOptions *options, char *line,
+@@ -1329,8 +1334,16 @@ process_server_config_line_depth(ServerOptions *options, char *line,
  	switch (opcode) {
  	/* Portable-specific options */
  	case sUsePAM:
 +#ifdef SET_USE_PAM
 +		/* UsePAM is always on and not configurable on Solaris */
```

Comment:
While you're here..change this to illumos?

Reply:
Sure why not?

Reply:
Oh, ummm, this patch actually from Oracle, so I'd prefer not to mess with it.

Reply:
Ok, I'm fine with that.